.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Add a brief and meaningful description. -->
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
+<!-- Run the command `ansible --version` and add the result here. -->
+* Project Version/Tag: (E.g release-1.0.1)
```

Comment:
Hi @sedroche, @aidenkeating ,

Is this the data/versions which are required for this project?  Please, could you give a hand here for we move forward?

Reply:
Hi @camilamacedo86 OS, oc and go versions should do

Reply:
Done 👍 tks could you approve for we move forward here.